DEVELOPER_GUIDE.md

```diff
@@ -20,9 +20,9 @@ You will need to install [node.js](https://nodejs.org/en/), [nvm](https://github
 
 ### Environment Setup
 
-1. Download OpenSearch for the version that matches the [OpenSearch Dashboards version specified in package.json](./package.json#L7).
-2. Download and install the appropriate [Security Analytics OpenSearch Plugin](https://github.com/opensearch-project/security-analytics).
-3. Download the OpenSearch Dashboards source code for the [version specified in package.json](./package.json#L7) you want to set up.
+1. [Download](https://opensearch.org/downloads.html) minimal distribution of OpenSearch for the version that matches the OpenSearch Dashboards version specified in [opensearch_dashboards.json](./opensearch_dashboards.json#L4).
```

Comment:
the non minimal one will have all the plugins installed and I have seen some errors related to a security plugin when running that package locally. Also, it would be unnecessary load to run other plugins.
That said, do you think it is better to use the complete one?

On another thought, we would need the notifications plugin if we want to test the alerts, so should also add point for installing that one too.

Reply:
I'm currently using the 2.4 branch of the github repo with the main branch of SAP, deployed on cloud desktop.  This has been working well, with no security related errors.  SAP is the only plugin I'm running.  Agreed on adding the notifications plugin.  I may be misunderstanding minimal vs complete, but I would be inclined to use what a typical user would install.

Reply:
We have had these instructions in the dev guide for some time now and I was just hoping to get them to a working state with minimal changes. I think we should have the minimal distribution because that gives us the basic functionalities and ability to install additional plugins that we would need. Just running the SAP backend would not be sufficient that way.

Reply:
Also these instructions are meant for developers and not end users of the product.

Reply:
Fair enough.